File: app/src/release/java/sk/vander/cmind/data/ReleaseDataModule.java
package sk.vander.cmind.data;

import dagger.Module;
import sk.vander.cmind.data.api.ReleaseApiModule;

/**
 * Created by arashid on 27/06/16.
 */
@Module(includes = {DataModule.class, ReleaseApiModule.class})
public class ReleaseDataModule {
}

File: app/src/release/kotlin/sk/vander/cmind/data/ReleaseDataModule.kt
package sk.vander.cmind.data

import dagger.Module
import sk.vander.cmind.data.api.ReleaseApiModule

/**
 * Created by arashid on 27/06/16.
 */
@Module(includes = arrayOf(DataModule::class, ReleaseApiModule::class))
class ReleaseDataModule
